test/hotspot/jtreg/runtime/appcds/cacheObject/ArchivedModuleWithCustomImageTest.java

```diff
@@ -25,15 +25,12 @@
  * @test
  * @summary Test archived module graph with custom runtime image
  * @requires vm.cds.archived.java.heap
- * @library /test/jdk/lib/testlibrary /test/lib /test/hotspot/jtreg/runtime/appcds
- * @modules java.base/jdk.internal.module
- *          java.management
- *          jdk.jlink
- *          jdk.compiler
+ * @requires vm.flagless
+ * @library /test/jdk/lib/testlibrary /test/lib /test/hotspot/jtreg/runtime/cds/appcds
  * @build sun.hotspot.WhiteBox
  * @compile CheckArchivedModuleApp.java
- * @run driver ClassFileInstaller -jar app.jar CheckArchivedModuleApp
- * @run driver ClassFileInstaller -jar WhiteBox.jar sun.hotspot.WhiteBox
+ * @run driver jdk.test.lib.helpers.ClassFileInstaller -jar app.jar CheckArchivedModuleApp
+ * @run driver jdk.test.lib.helpers.ClassFileInstaller -jar WhiteBox.jar sun.hotspot.WhiteBox
```

Comment:
Includes the changes from [JDK-8263549](https://bugs.openjdk.org/browse/JDK-8263549) ( commit [a7aba2b](https://github.com/openjdk/jdk/commit/a7aba2b61cbe937a1aa59636f6e28afdec1e10d0) ) on this file